### PR TITLE
Implement stupid computer move

### PR DESCRIPTION
**Pull request by merecarvill**

### CODE REVIEW

File: spec/minimax_server/core_spec.clj [hunk 1]
```diff
@@ -1,8 +1,8 @@
 (ns minimax-server.core-spec
-  (:import )
-  (:require [speclj.core :refer :all]
+  (import [scarvill.httpserver.response ResponseBuilder])
+  (require [speclj.core :refer :all]
             [minimax-server.core :refer :all]))
 
 (describe "temporary tests"
```
Comment (ericmeyer) [inline spec/minimax_server/core_spec.clj:6]: `"temporary tests"` :(

File: spec/minimax_server/core_spec.clj [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
   (it "can use things from http server jar"
```
Comment (ericmeyer) [inline spec/minimax_server/core_spec.clj:7]: Does this test still have value?

File: src/minimax_server/computer_player.clj [hunk 3]
```diff
@@ -0,0 +1,7 @@
+(ns minimax-server.computer-player)
+
+(defn zip-indexes [board]
+  (map vector (range (count board)) board))
+
+(defn stupid-move [board]
+  (first (first (filter (fn [[index mark]] (= :_ mark)) (zip-indexes board)))))
```
Comment (ericmeyer) [inline src/minimax_server/computer_player.clj:7]: Am I missing something, or is this computer not random?

**Comment by ericmeyer** [comment]
Good job splitting the two commits, and also from breaking this work off from the router that uses it.

File: src/minimax_server/computer_player.clj (hunk 3 above)
Comment (aaronlahey) [inline src/minimax_server/computer_player.clj:7]:
``` clojure
;; you can pass two sequences to mapcat and get a simpler implementation here...
(defn available-locations
  [board]
  (mapcat #(when (= :_ %1) [%2])
          board
          (range)))

;; the first available location
(first (available-locations board))

;; or if you want a random move...
(rand-nth (available-locations board))

;; or if you want to use transducers which are quite neat...
(def available-locations
  (comp (map-indexed vector)
        (filter (comp (partial = :_) last))
        (map first)))

;; take the first value from a lazy sequence of available locations...
(first (sequence available-locations board))

;; put all available locations in a vector and then choose a random one...
(rand-nth (into [] available-locations board))
```

**Pull request merged**